apps/api/src/domain/models/playlist/entities/playlist.entity.ts

```diff
@@ -67,6 +68,16 @@ export class Playlist extends Resource {
         });
     }
 
+    translateName(textItem: MultilingualTextItem) {
+        if (this.name.items.some(({ languageCode }) => languageCode === textItem.languageCode))
+            return new DuplicateLanguageInMultilingualTextError(textItem.languageCode);
+        // TODO validate the item
```

Comment:
Remove comment. This isn't necessary as `safeClone` runs full invariant validation.

Reply:
done